site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -193,4 +264,25 @@ This is particularly true for a zonal GKE control plane, which can go down tempo
 
 ## Next Steps
 
-- Continue to [Install Agones]({{< relref "../Install Agones/_index.md" >}}).
+### GKE Standard
+
+If you used a GKE Standard cluster, continue to [Install Agones]({{< relref "../Install Agones/_index.md" >}}).
+
+### GKE Autopilot
+
+If you used a GKE Autopilot cluster:
+
+1.  [Install Helm](https://helm.sh/docs/intro/quickstart/) 
+1.  Install Agones:
+
+    ```bash
+    helm repo add agones https://agones.dev/chart/stable
+    helm repo update
+    helm install my-release \
+      --namespace agones-system \
+      --create-namespace agones/agones \
+      --set agones.featureGates="SplitControllerAndExtensions=true"
```

Comment:
Do we want to be clear in this section "Since this is currently alpha support, we need to enable an Alpha feature to have Agones work on Autopilot. Once this feature has been promoted, this will no longer be neccessary"

(or something like that?)